src/gwt/src/org/rstudio/studio/client/application/ApplicationThemes.java

```diff
@@ -42,17 +46,41 @@ public class ApplicationThemes implements ThemeChangedEvent.Handler,
    public ApplicationThemes(Provider<UserPrefs> pUserPrefs,
                             Provider<UserState> pUserState,
                             Provider<AceThemes> pAceThemes,
-                            EventBus events,
-                            ThemeServerOperations server)
+                            Session session,
+                            EventBus events)
    {
       userPrefs_ = pUserPrefs;
       userState_ = pUserState;
       pAceThemes_ = pAceThemes;
       events_ = events;
-      server_ = server;
+      
+      themeColors_ = JsObject.createJsObject();
       
       events_.addHandler(ThemeChangedEvent.TYPE, this);
       events_.addHandler(ComputeThemeColorsEvent.TYPE, this);
+      
+      // NOTE: These values are read by the session, so if any names need to change
+      // here they should be synchronized on the session side as well.
```

Comment:
Maybe it's worth mentioning `SessionThemes.cpp` here so that we know where to look. Although maybe not, a simple search for "themeInfo" would suffice.

Reply:
Oops; merged before seeing this. Updated on main: https://github.com/rstudio/rstudio/commit/fc3f77a63db11e0a931973f085e90ea495500af9